brave/brave-core
# perf(wallet): do not wait for sign tx reqs when initing panel

Resolves https://github.com/brave/brave-browser/issues/37401
Resolves https://github.com/brave/brave-browser/issues/37402



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Open https://pwgoom.csb.app/
2. connect a Solana account
3. click "Sign Transaction"
4. panel should open to the sign transaction panel
5. close the panel
6. reopen the panel
7. panel should re-open to the sign transaction panel
8. Approve or reject the transaction
9. panel should close if no additional signatures are being requested

Repeat the above with a hardware account
